js/scenes/ProjectBrief/ProjectBriefContainer.js

```diff
@@ -12,9 +12,10 @@ class ProjectBriefContainer extends Component {
     }
   };
   componentDidMount() {
-    this.props.dispatch(getProjectDeliverables("deloitte", "blockchain"));
-    this.props.dispatch(getCompanyProjects("deloitte", "blockchain"));
-    this.props.dispatch(getCompany("deloitte"))
+
+    this.props.dispatch(getProjectDeliverables(this.props.route.params.props.company, this.props.route.params.props.id));
+    this.props.dispatch(getCompanyProjects(this.props.route.params.props.company, this.props.route.params.props.id));
+    this.props.dispatch(getCompany(this.props.route.params.props.company))
```

Comment:
you can deconstruct and just pass company instead of doing this.props.route.params.props.company.